tasks/process_agent.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+import os
+import shutil
+import sys
+
+from invoke import task
+from subprocess import check_output
+
+from .utils import bin_name, get_build_flags, REPO_PATH, get_version, get_git_branch_name, get_go_version, get_git_commit, get_version_numeric_only
+from .build_tags import get_default_build_tags
+
+BIN_DIR = os.path.join(".", "bin", "process-agent")
+BIN_PATH = os.path.join(BIN_DIR, bin_name("process-agent", android=False))
+
+
+@task
+def build(ctx, race=False, incremental_build=False, use_embedded_libs=False, puppy=False, use_venv=False):
+    """
+    Build the process agent
+    """
+
+    # generate windows resources
+    if sys.platform == 'win32':
```

Comment:
Have you done any testing around the windows binary built by this yet?

Reply:
Not yet I don't really have a good windows environment for this, but I will test the binaries built from the regular pipeline in the next PR 👍